src/renderer/App.js

```diff
@@ -18,6 +18,7 @@
 import "@api/colormap";
 import Focus from "@api/focus";
 import "@api/keymap";
+import "@api/macros";
```

Comment:
I suspect that this import is to get the macros commands registered in Focus? is that right? Given that we always want to do that, why not just have @api/focus do that import?

Reply:
Historical reasons, mostly, but they no longer apply, really. Would make sense to merge `colormap` and `macros` into focus, indeed.